### PR TITLE
implementation of nndetector

### PR DESCRIPTION
**Pull request by Hyoungsukim**
- object detection and learning personal object on mobile

**Self evaluation:**
1. Build test:     [X]Passed [ ]Failed [ ]Skipped
2. Run test:     [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by myungjoo — COMMENTED** [review]
Please add SPDX license identifier for all source files (*.c,*.cpp,*.h,*.kt, ...)

**Comment by Hyoungsukim** [comment]
> Please add SPDX license identifier for all source files (_.c,_.cpp,_.h,_.kt, ...)

I've done to add SPDX license identifier.

**Review by myungjoo — APPROVED** [review]

File: Applications/Android/NNDetector/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# NNDetector
+NNDetector based on nntrainer is an demo application conducting object detection.
```
Comment (wooksong) [inline Applications/Android/NNDetector/README.md:2]:
```suggestion
NNDetector is a demo application conducting object detection based on nntrainer.
```

File: Applications/Android/NNDetector/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+You can inference an object and train with your personal items. Below is showing how to detect and train with NNDetector.
```
Comment (wooksong) [inline Applications/Android/NNDetector/README.md:3]: 'inference' is a noun.

```suggestion
You can detect an object and train with your personal items. Below is a showing of how to detect and train with NNDetector.
```

File: Applications/Android/NNDetector/app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    }
+    packagingOptions {
+        resources {
+            excludes += '/META-INF/{AL2.0,LGPL2.1}'
+        }
+    }
+}
+
+dependencies {
+
```
Comment (wooksong) [inline Applications/Android/NNDetector/app/build.gradle:52]:
```suggestion
```

File: Applications/Android/NNDetector/app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            android:name=".MainActivity"
+            android:launchMode="singleTask"
+            android:exported="true"
+            android:label="@string/app_name"
+            android:theme="@style/Theme.NNDetector">
+<!--            <intent-filter>-->
+<!--                <action android:name="android.intent.action.MAIN" />-->
+
+<!--                <category android:name="android.intent.category.LAUNCHER" />-->
+<!--            </intent-filter>-->
```
Comment (wooksong) [inline Applications/Android/NNDetector/app/src/main/AndroidManifest.xml:45]:
```suggestion
```

**Review by wooksong — COMMENTED** [review]
```*.tflite``` files look empty.

**Comment by myungjoo** [comment]
I recommend to put large files (such as .tflite) at somewhere else (other repos) and let it loaded from the Internet at built time or install time.

**Comment by Hyoungsukim** [comment]
I will reorganize codes and upload, again.

**Comment by myungjoo** [comment]
You can keep a PR opened and update codes with ```git push {your fork} {your branch} --force```.
Please do not close and open another for the same topic.

**Comment by Hyoungsukim** [comment]
reopen with empty code.

**Pull request closed without merging**